recipes/pulsar_phase/pulsar_phase_computation.ipynb

```diff
@@ -0,0 +1,1895 @@
+{
```

Comment:
<p>The file seems to be created by alvarom on its mac. It seems to be in contradiction with the fact that the ephemeris file is from Fermi-LAT.</p><p>There is a kind of contradiction between the header saying that this is "created" and the fact that the ephemeris is from Kerr et al.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy-recipes/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>

Reply:
It actually is the model that is created by `pint.models.get_model(ephemeris_file)`which is created not the actual ephemeris file. But I understand that this is confusing